subcommittees/mentoring/ITProposal.md

```diff
@@ -0,0 +1,114 @@
+# Suggestions to Instructor Training
+
+**This is a rough draft.**
+
+## Terminology
+
+- [Checkout session](#checkout-session): 1h meeting where learners at
+  Instructor Training deliver and review five-minute lessons
+
+  The meetings are lead by Instructor Trainers and can have at maximum
+  6 learners.
+- Mini-lesson: 1h lesson for at least 3 people delivering part of one
+  lesson from Software or Data Carpentry curriculum.
+- Session: 3h lesson for at least 10 people delivering one full lesson
+  from [Software][swc-lessons] or [Data Carpentry][dc-lessons] curriculum.
+- Workshop: 12h lesson for at least 20 people delivering the full
+  curriculum of [Software][swc-core] or Data Carpentry.
+- Instructor in Training: can teach and organize a mini-lesson,
+  session, or workshop under the supervision of a badged Software or
+  Data Carpentry instructor.
+- Instructor: can lead, teach, and organize a mini-lesson, session, or
+  workshop.
+- Program Coordinator: TODO (is this the Executive Director?)
+
+## Badges
+
+Anyone interested in joining the Software and/or Data Carpentry team
+of instructors **must** complete the Instructor Training in any format
+offered by Software and/or Data Carpentry.
+
+Software and Data Carpentry will assign "Instructor in Training"
+badges to people who have successfully completed a [checkout
+session](#checkout-session).  After badging, Instructors in Training
+have one year to teach one mini-lesson, one session, or one workshop
```

Comment:
We are yet to see the outcomes of our Autumn 2015 pilot Instructor Training (in which we were encouraging the commitment to teach at full workshops within x amount of time), however, I would try to propagate something similar in the ideas discussed here. That is I'd suggest splitting that and setting a shorter timeline for mini-lesson and then a longer for session / workshop.